docs/content_management/forms/form_builder_guide.md

```diff
@@ -13,6 +13,8 @@ By adding forms on the website, you can increase its functionality and improve u
 Use Form Builder to create various forms, such as survey, questionnaire, sign-up form, using basic form fields available in the Form Builder.
 You can also manage your forms and review the results gathered from the website users.
 
+<!--ARCADE EMBED START--><div style="position: relative; padding-bottom: calc(51.27314814814815% + 41px); height: 0; width: 100%;"><iframe src="https://demo.arcade.software/6TuwPfbDTXTo6SnYSiyW?embed&embed_mobile=tab&embed_desktop=inline&show_copy_link=true" title="Form Builder interface" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; color-scheme: light;" ></iframe></div><!--ARCADE EMBED END-->
```

Comment:
Funny thing happens when you hover over the play button:
![image](https://github.com/user-attachments/assets/1d1058e7-c4bc-43ff-a111-7018cc04ff7c)

Reply:
I don't see this behaviour.